Remove references to wazuh.yml file and adapt to the new configuration system (#7050)
wazuh/wazuh-documentation: source/user-manual/wazuh-dashboard/config-file.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::
   :description: The Wazuh dashboard includes configuration options to define custom settings. Learn more about it in this section.

.. _wazuh_dashboard_config_file:

Configuration
=============

You can configure the Wazuh dashboard and plugins as follows:

- **Wazuh dashboard settings**: You can edit the Wazuh dashboard configuration file.

   -  ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` for installations from packages.
   -  ``/usr/share/wazuh-dashboard/config/opensearch_dashoards.yml`` for Docker installations.

> - **Plugins settings**: Users with privileges to manage the Wazuh indexer security REST API can manage the plugins settings from the Wazuh dashboard. Wazuh stores the custom values in a saved object.

Author: I applied this comment as a review suggestion: https://github.com/wazuh/wazuh-documentation/pull/7050/commits/cc7ec0b507bccdbaea494de04f3f0ed0d66fca04#r1523495979

   -  **Dashboard management** > **App Settings**: To customize several settings.
   -  **Dashboard management** > **Server APIs**: To customize API host entries.

> This section describes all the settings available for each location.

Author: Plugins settings and Dashboard settings.

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::

>   :description: The Wazuh dashboard includes a configuration where you can define custom values for several options. Learn more about it in this section.

Reviewer: ```suggestion
   :description: The Wazuh dashboard includes configuration options to define custom settings. Learn more about it in this section.
```

.. _wazuh_dashboard_config_file:

Configuration
=============

> The configuration related to Wazuh plugins can be defined in:

Reviewer: ```suggestion
You can configure the Wazuh dashboard and plugins as follows:
```

Reviewer: We use active voice as much as possible

> - **Plugins settings**:

Reviewer: -  **Wazuh dashboard settings**: You can edit the Wazuh dashboard configuration file.

   -  ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` for installations from packages.
   -  ``/usr/share/wazuh-dashboard/config/opensearch_dashoards.yml`` for Docker installations.

-  **Plugins settings**: Users with privileges to manage the Wazuh indexer REST API can manage the plugins settings from the Wazuh dashboard. Wazuh stores the custom values in a saved object.

   -  **Dashboard management** > **App Settings**: To customize several settings.
   -  **Dashboard management** > **Server APIs**: To customize API host entries.

Author: Applied the suggestion on: 7211ef37558ea73c7b1dd235ed7dfe7e2e23f6bb
